shell/browser/api/electron_api_desktop_capturer.cc

```diff
@@ -181,6 +181,42 @@ DesktopCapturer::DesktopCapturer(v8::Isolate* isolate) {}
 
 DesktopCapturer::~DesktopCapturer() = default;
 
+DesktopCapturer::DesktopListListener::DesktopListListener(
+    OnceCallback update_callback,
+    OnceCallback failure_callback,
+    bool skip_thumbnails) {
+  update_callback_ = std::move(update_callback);
+  failure_callback_ = std::move(failure_callback);
+  have_thumbnail_ = skip_thumbnails;
+}
+
+DesktopCapturer::DesktopListListener::~DesktopListListener() = default;
+
+void DesktopCapturer::DesktopListListener::OnDelegatedSourceListSelection() {
+  if (have_thumbnail_) {
+    std::move(this->update_callback_).Run();
```

Comment:
We don't need to use `this`:

```suggestion
    std::move(update_callback_).Run();
```